src/exml_nif.erl

```diff
@@ -11,11 +11,11 @@
 -type parser() :: term().
 -type stream_element() :: exml:element() | exml_stream:start() | exml_stream:stop().
 
--export([create/2, parse/1, parse_next/2, escape_cdata/1,
-         to_binary/2, reset_parser/1]).
+-export([create/2, parse/1, parse_next/2, escape_cdata/1, to_binary/2, reset_parser/1]).
```

Comment:
But this list of exports is flattened to one line. Why? Is there a heuristic that if they're one function per line then they're left as is, but otherwise they're compacted to just one line?

Reply:
There is a configuration parameter: `inline_items` that defines which lists are inlined and which ones are not. The default value is `{when_over, 25}` and, since this list is longer than 25 elements, it's inlined.

Reply:
Are you using `inline_items => true` in your options, by any chance?
If not, then this is a bugs and you should report it in the rebar3_format repo.

Reply:
I opened the issue here: AdRoll/rebar3_format#88